src/components/LanguagePicker/LanguagePicker.tsx

```diff
@@ -7,22 +7,17 @@ import { ButtonGroup, CtaButton } from '../Button/index';
 import { verticalScale, scale } from '../../utils/index';
 import { withNavigation } from 'react-navigation';
 import { Content } from '../AppShell';
+import covidIcon from '../../assets/images/covid-icon.png';
 
-const AllLanguages = styled.View`
-  flex: 1;
-`;
+const LanguageScreen = styled.View``;
 
-const Language = styled<{ index: number }>(Content)`
-  background: ${({ index }) =>
-    index % 2 ? Colors.backgroundAlt : 'transparent'};
-  flex: 1;
+const Title = styled(Content)`
+  height: ${verticalScale(150)};
 `;
 
-const LanguageContent = styled.View`
-  justify-content: space-between;
-  flex: 1;
-  padding-vertical: ${verticalScale(8)};
-`;
+const LanguageList = styled(Content)``;
+
+const Continue = styled(Content)``;
```

Comment:
Could we instead wrap everything in one `<Content />` element?

Reply:
Please, refer to this comment: https://github.com/aranja/rakning-c19-app/pull/51#discussion_r407065427